framework/src/reporters/ReporterData.C

```diff
@@ -52,6 +52,55 @@ ReporterData::getPostprocessorNames() const
   return output;
 }
 
+DenseVector<Real>
+ReporterData::getAllRealReporterValues() const
+{
+  DenseVector<Real> all_values;
+
+  std::vector<Real> & output = all_values.get_values();
+
+  for (const auto & name_context_pair : _context_ptrs)
+  {
+    const ReporterName & rname = name_context_pair.first;
+
+    if (hasReporterValue<Real>(rname))
+      output.push_back(getReporterValue<Real>(rname.getCombinedName()));
+
+    if (hasReporterValue<std::vector<Real>>(rname))
+    {
+      const auto & vec = getReporterValue<std::vector<Real>>(rname.getCombinedName());
+      for (const auto & v : vec)
+        output.push_back(v);
+    }
```

Comment:
Would be nice to include a vector postprocessor in your test in order to hit these lines but not critical

Reply:
Will just turn on this flag for a vector postprocessor test.